test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -192,6 +261,1427 @@ public void testCreateDeleteKeyValueTable() {
+        List<TableEntry<Integer, String>> entryList2 = new ArrayList<>();
+        String[] valueArray2 = new String[10];
+        for (int i = 0; i < valueArray1.length; i++) {
+            valueArray2[i] = getValue();
+        }
+        for (int i = 0; i < keyArray.length; i++) {
+            entryList2.add(TableEntry.versioned(keyArray[i], versionList1.get(i), valueArray2[i]));
+        }
+        log.info("Entry list2: {}", entryList2);
+
+        // Conditionally updating the values of all the Keys with the specified key version and KeyFamily.
+        List<Version> versionList2 = newKeyValueTable.replaceAll(keyFamily, entryList2).join();
+        assertNotNull("Version list2 should not be empty", versionList2);
+        assertEquals("Version list2 size should be same as entry list2 size",
+                versionList2.size(), entryList2.size());
+        log.info("Version list2: {}", versionList2);
+
+        log.info("Key list: {}", keyList);
+        List<TableEntry<Integer, String>> getEntryList2 = newKeyValueTable.getAll(keyFamily, keyList).join();
+        assertNotNull("Get Entry List2 should not be empty", getEntryList2);
+        assertEquals("Get Entry List2 size should be same as get entry list2 size",
+                getEntryList2.size(), entryList2.size());
+        log.info("Get Entry List2: {}", getEntryList2);
+        for (int i = 0; i < keyArray.length; i++) {
+            assertEquals("Corresponding key for getEntryList should be as inserted",
+                    keyList.get(i), getEntryList2.get(i).getKey().getKey());
+            assertEquals("Corresponding key for entryList should be as inserted",
+                    entryList2.get(i).getKey().getKey(), getEntryList2.get(i).getKey().getKey());
+        }
+
+        // Comparing each & every entry values of both entrylist and getEntryList.
+        for (int i = 0; i < keyArray.length; i++) {
+            assertEquals("All string values of both entryList2 and getEntryList2 should be equal",
+                    entryList2.get(i).getValue(), getEntryList2.get(i).getValue());
+        }
+
+        // Comparing each & every version values of both versionList1 and versionList2.
+        for (int i = 0; i < keyArray.length; i++) {
+            assertNotEquals("All version values of both getEntryList1 and getEntryList2 should not be equal",
+                    getEntryList1.get(i).getKey().getVersion(), getEntryList2.get(i).getKey().getVersion());
+            assertNotEquals("All version values of both versionList1 and versionList2 should not be equal",
+                    versionList1.get(i), versionList2.get(i));
+        }
+        log.info("Successfully completed test Conditional update multiple Key values with KF using its version");
+    }
+
+    /**
+     * Test case-49: Conditional delete multiple TableEntries with KeyFamily
+     * using its version
+     */
+    @Test
+    public void testConditionDeleteMultiKeys() {
```

Comment:
This is already covered in the base class.

Reply:
Sure, I'll remove this test as its a duplicate.